src/connection.ts

```diff
@@ -95,6 +95,7 @@ export async function getPostgresParams() {
         }
         dbConnParams['password'] = await keytar.getPassword('vscode.vscode-sql', activeConn)
     }
+    dbConnParams['ssl'] = vscode.workspace.getConfiguration(`vscodeSql.connections.${activeConn}`).get('ssl')
```

Comment:
get the `ssl` param directly from user settings for both db-facts and normal connections